LiteFarmOrg/LiteFarm
# LF-2431 Task status filter needs to be dynamic

Ticket: [LF-2431](https://lite-farm.atlassian.net/browse/LF-2431)

- Made the keys in filter.json consistent for each language
- Fixed language translation issue with task status filter

To test:
- Change to a language other than english
- Go to tasks page -> Task filters page
- Make sure that status filters show up in that language
